src/Bicep.Local.Extension/Rpc/GrpcChannelHelper.cs

```diff
@@ -48,7 +48,7 @@ public static GrpcChannel CreateChannel(string socketPath)
         };
 
         // The URL is not used, but it must be a valid URI.
```

Comment:
What is this URL needed for then?

Reply:
It's parsed initially by the `GrpcChannel` API (which throws if it's invalid), but the actual socket connection logic is overridden by setting the `HttpHandler`